clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -145,6 +145,33 @@ void check_ast()
   [[intel::simple_dual_port]] int var_dual_port;
   [[intel::force_pow2_depth(1)]] int var_force_p2d;
   [[intel::force_pow2_depth(1)]] const int const_force_p2d[64] = {0, 1};
+
+  // Checking of duplicate argument values.
+  //CHECK: VarDecl{{.*}}var_max_replicates
```

Comment:
Actually sorry I just noticed this. For duplicate arguments should we be adding the attribute twice? This may be outside the scope of this PR because I see @AaronBallman did the same thing in #3224. Is there a reason we don't just silently ignore duplicate attributes?

Reply:
Good catch! That's a mistake on my part -- we should silently drop a duplicate attribute when adding or merging.

Reply:
I have created https://github.com/intel/llvm/issues/3336 and will fix the problem in next patch.

Thanks @elizabethandrews for catching this.